readme.adoc

```diff
@@ -583,7 +583,20 @@ Make sure `kubectl` CLI is installed and configured for the Kubernetes cluster.
 
 === Deployment: Kubepack
 
-https://github.com/aws-samples/aws-microservices-deploy-options/issues/63
+Make sure `kubectl` and `kubepack` CLI is installed and configured for the Kubernetes cluster. To install `kubepack` cli, run:
+
+```
+wget -O kubepack https://github.com/kubepack/kubepack/releases/download/0.1.0/kubepack-darwin-amd64 \
```

Comment:
URL is incorrect, it should be https://github.com/kubepack/pack/releases/download/0.1.0/pack-darwin-amd64